src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1500,7 +1500,21 @@ def aks_kollect(cmd,    # pylint: disable=too-many-statements,too-many-locals
     else:
         display_diagnostics_report(temp_kubeconfig_path)
 
-    return
+
+def _trim_fqdn_name_containing_hcp(normalized_fqdn: str) -> str:
```

Comment:
Can we please move to this to a helper file

Reply:
@gtracer Cool, sounds good, i have moved this to helper:

* Moving fwd plan is to refactor this file. Essentially, I have a plan to do another PR with some refactor for `custom.py` move rest of the helper `functs` out.